stable/datadog/templates/daemonset.yaml

```diff
@@ -0,0 +1,56 @@
+{{- if .Values.datadog.apiKey }}
+apiVersion: extensions/v1beta1
+kind: DaemonSet
```

Comment:
Would it make sense to enable Service Discovery with the Docker backend as well? And possible add a dependency on kube-state-metrics?